### DISCUSSION TITLE
Quarto and Julia: any experiences, examples or news in this space?

### DISCUSSION THREAD

**Discussion by pat-alt** [quarto-dev/quarto-cli#3345] [Q&A]
I'll be talking about Quarto + Julia at a Julia Meetup in Eindhoven next week and am interested in hearing about other people's experiences and/or any news in this space. If you have any interesting examples you'd like me to highlight, that would also be useful. 

In case you want to have a look at what I currently have, here are the preliminary [slides](https://www.paltmeyer.com/content/talks/posts/2022-julia-eindhoven/presentation.html#/title-slide). 

Any thoughts/comments would be much appreciated! 

Thanks 🙏🏽

**Comment by chrisvwx** [comment]
One thing that you might note is that some Julia users (like me) who are not experienced with Python may have problems getting even the [basic example](https://quarto.org/docs/computations/julia.html) working.  I think I've been spoiled by Julia's excellent Pkg system, including the packaged binaries.

**Reply by pat-alt** [reply]
Thanks, `Pkg` is indeed amazing!

I have a [slide](https://www.paltmeyer.com/content/talks/posts/2022-julia-eindhoven/presentation.html#/vscode-quarto-and-julia) on getting started where I will recommend adding `IJulia` to `startup.jl`. Planning to also point out that `IJulia` needs to be rebuild every time you update Julia. 

I don't see where the basic example depends on experience with Python though? Where exactly do you see potential problems?

**Reply by chrisvwx** [reply]
> I don't see where the basic example depends on experience with Python though? 

Quarto itself seems to depend on python, and I haven't got it to properly work with Julia. I have three versions of Python installed (MacOS, Homebrew, and one that Julia itself installed), which certainly doesn't help.  I'm still [working](https://github.com/quarto-dev/quarto-cli/discussions/2606) on things and will get it figured out.  I guess most people have more experience with Python than I do. I was frustrated when I saw this thread, which triggered my comment above. You can ignore it

**Comment by camilogarciabotero** [comment]
I wrote some thoughts about that some weeks ago in this discussion: https://github.com/quarto-dev/quarto-cli/discussions/2506, but summarizing, I believe is pretty nice and smooth!
